src/admin-views/editor/panel/ticket.php

```diff
@@ -23,11 +23,38 @@
 
 		$ticket = $provider->get_ticket( $post_id, $ticket_id );
 	}
+
+	if ( ! empty( $ticket->start_date ) ) {
+		$start_date = Tribe__Date_Utils::date_only( $ticket->start_date, false, $datepicker_format );
+	} else {
+		$start_date = null;
+	}
+
+	if ( ! empty( $ticket->end_date ) ) {
+		$end_date = Tribe__Date_Utils::date_only( $ticket->end_date, false, $datepicker_format );
+	} else {
+		$end_date = null;
+	}
 }
 
 $modules = Tribe__Tickets__Tickets::modules();
 
 $default_module_class = empty( $default_module ) ? '' : $default_module->class_name;
+
+$datepicker_format = Tribe__Date_Utils::datepicker_formats( Tribe__Date_Utils::get_datepicker_format_index() );
+
+$timepicker_step = 30;
+
+if ( class_exists( 'Tribe__Events__Main' ) ) {
+	$timepicker_step = (int) tribe( 'tec.admin.event-meta-box' )->get_timepicker_step( 'start' );
+}
+
+$timepicker_round = '00:00:00';
+
+$start_date_errors = [
+		'is-required'         => __( 'Start sale date cannot be empty.', 'event-tickets' ),
```

Comment:
:no_entry_sign: Array item not aligned correctly; expected 4 spaces but found 8

<sub>hash: 1ed037559dccf3c351ab53646eb80c7e</sub>